core/plugins/org.polarsys.capella.core.platform.sirius.sirius.validation/src/org/polarsys/capella/core/platform/sirius/sirius/validation/ddiagram/ILinkModifierHandler.java

```diff
@@ -0,0 +1,7 @@
+package org.polarsys.capella.core.platform.sirius.sirius.validation.ddiagram;
+
+
+public interface ILinkModifierHandler {
```

Comment:
Document this interface (e.g. used in a description parser, react to link encountering by update the link with new info)